test/horde_supervisor_test.exs

```diff
@@ -38,7 +38,7 @@ defmodule HordeSupervisorTest do
 
   describe ".start_child/2" do
     test "starts a process", context do
-      Horde.Supervisor.start_child(context.horde_1, context.task_def)
+      assert {:ok, _pid} = Horde.Supervisor.start_child(context.horde_1, context.task_def)
```

Comment:
if we know the `pid`, would be a nice extra check it's also the one we receive at [line 43](https://github.com/derekkraan/horde/pull/3/files#diff-a8d0ac8f48d80df61cea093956142e33R43) ?